js/deps.ts

```diff
@@ -1,24 +1,24 @@
-import React, { Fragment } from "https://esm.sh/react@17.0.2";
-import ReactDOM from "https://esm.sh/react-dom@17.0.2";
+import React, { Fragment } from "https://cdn.skypack.dev/react@17.0.2";
+import ReactDOM from "https://cdn.skypack.dev/react-dom@17.0.2";
 
 export {
   BrowserRouter,
   Link,
   NavLink,
   Route,
-  Switch,
-} from "https://esm.sh/react-router-dom@5.2.0";
+  Routes,
+} from "https://cdn.skypack.dev/react-router-dom@6.1.1";
```

Comment:
In my local environment, I couldn't load the type definition of `react-router-dom` correctly from skypack with `?dts` appended (I tried several versions, but I couldn't find any working version)

If I switched to [wouter](https://github.com/molefrog/wouter), it worked at least with my basic example. (I use at least `<Route>`, `<Link>`, and `<Switch>`)

Reply:
I'll try Wouter before giving up on react with typescript.